fastlane/Constants/Constant.swift

```diff
@@ -28,12 +28,20 @@ enum Constant {
     static let appleProductionTeamId = "<#teamId#>"
     static let keychainName = "{PROJECT_NAME}_keychain"
     static let matchURL = "git@github.com:{organization}/{repo}.git"
-    static let apiKey: [String: Any] = [
-        "key_id" : Secret.appStoreKeyIdKey,
-        "issuer_id": Secret.appStoreIssuerIdKey,
-        "key": Secret.appstoreConnectAPIKey,
-        "in_house": false
-    ]
+    static let apiKey: [String: Any] = {
+        var key = Secret.appstoreConnectAPIKey
+        if let data = Data(base64Encoded: Secret.appstoreConnectAPIKey),
+           let decodedKey = String(data: data, encoding: .utf8) {
+            key = decodedKey
+        }
```

Comment:
Because the API key is stored in CI's secret, why do we need to encode it and decode it again? I think it is redundant to encode the key, it is not like the `.env` file.

Reply:
@phongvhd93 it's because we are using it as base64 🙏 

![CleanShot 2024-07-03 at 10 27 30](https://github.com/nimblehq/ios-templates/assets/17875522/ac53efad-7133-40c2-ba79-3dc51fad3ac5)